src/main/java/com/autotune/analyzer/recommendations/engine/RecommendationConstants.java

```diff
@@ -0,0 +1,2628 @@
+package com.autotune.analyzer.recommendations.engine;
```

Comment:
This appears to be mistake. We already have this data in the `RecommendationEngine` class. 
We also have the `RecommendationConstants` class as well. 
Please check and revert this change.